src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -110,6 +110,34 @@ SLuint32 AudioOutputStreamOpenSLES::channelCountToChannelMask(int channelCount)
 }
 
 Result AudioOutputStreamOpenSLES::open() {
+    // Log unsupported attributes
+    // only report if changed from the default
+    // Usage::Media
+    // NOT TRUE! See OpenSLES_convertOutputUsage
+    // if (mUseage != Usage::Media) {
+    //    LOGW("Usage not supported on OpenSL ES streams.");
+    //}
+
+    // Content Type
+    if (mContentType != ContentType::Music) {
+        LOGW("ContentType [AudioStreamBuilder::setContentType()] "
+             "is not supported on OpenSLES output streams.");
+    }
+    // Sharing Mode
+    if (mSharingMode != SharingMode::Shared) {
+        LOGW("SharingMode [AudioStreamBuilder::setSharingMode()] "
+             "is not supported on OpenSLES output streams.");
```

Comment:
remove "output"

Reply:
You're OK with the verbosity of this message then, i.e the reference to the setter function?

Reply:
Sure, the purpose of these logs is to help the developer. Being a little verbose is OK.